rest_framework_api_key/admin.py

```diff
@@ -14,7 +14,7 @@ class APIKeyAdmin(admin.ModelAdmin):
     readonly_fields = ("get_api_key",)
     search_fields = ("name",)
 
-    fieldsets = ((None, {"fields": ("name", "revoked", "get_api_key")}),)
+    fieldsets = ((None, {"fields": ("name", "revoked", "get_api_key", 'expiry_date')}),)
```

Comment:
Should we place the `expiry_date` just after the `name`? They are the only two fields that we generally want to define when creating an API key, so I guess it would make sense to place them side to side.